### ISSUE TITLE
Compile changes in pdf.js

### ISSUE THREAD

**Issue by Gibran-16** [mozilla/pdf.js#7682]
Hello,

I did some changes in pdf.js, I customized something in html, add a js file, add bootstrap and jquery modules, now I need to compile pdf.js but it do not include my changes, I executed node make generic, how could compile my changes?

Thanks

**Comment by yurydelendik** [comment]
It's hard to tell without seeing your changes in /web/*.html files. Pay attention that some of the code is replaced by the preprocessor, so pay attention to the `<!--#if` instructions. Closing as answered.

**Comment by Gibran-16** [comment]
Thanks for your answer, I did changes in /web/* folder, there I added one css files and one js file that I call in viewer.html, I also added jquery, jqueryiu and bootstrap with npm install xxx --save. In the viewer.html I added some html for the bootstrap modal, It's ok, the problem is with bootstrap and jquery that do not find the resources, when I compile it doesn't add those files.
**Failed to load resource: net::ERR_FILE_NOT_FOUND

Thanks.
